### ISSUE TITLE
unexpected keyword argument when load qwen 2.5 instruct

### ISSUE THREAD

**Issue by blizzardwj** [xorbitsai/inference#2420]
### System Info / 系統信息

- Description:    Ubuntu 20.04.6 LTS
  nvidia-cuda-runtime-cu12  12.1.105                 pypi_0    pypi  
- llama-cpp-python          0.2.90                   pypi_0    pypi
  opencv-contrib-python     4.10.0.84                pypi_0    pypi  
  python                    3.10.14         hd12c33a_0_cpython    conda-forge  
  python-dateutil           2.9.0.post0              pypi_0    pypi  
  python-dotenv             1.0.1                    pypi_0    pypi  
  python-jose               3.3.0                    pypi_0    pypi  
  python-multipart          0.0.9                    pypi_0    pypi  
  quantile-python           1.1                      pypi_0    pypi  
  sentence-transformers     3.2.0                    pypi_0    pypi  
  transformers              4.45.2                   pypi_0    pypi  
  vllm                      0.5.2+cu125              pypi_0    pypi  
  vllm-flash-attn           2.5.9.post1              pypi_0    pypi

### Running Xinference with Docker? / 是否使用 Docker 运行 Xinfernece？

- [ ] docker / docker
- [X] pip install / 通过 pip install 安装
- [ ] installation from source / 从源码安装

### Version info / 版本信息

0.15.3

### The command used to start Xinference / 用以启动 xinference 的命令

xinference-local --host 0.0.0.0 --port 9997

### Reproduction / 复现过程

It is fine to load qwen 2.5 instruct in ver. 0.15.2, but in 0.15.3 I encounter the following:
1. load the model like

3. then prompt
```
2024-10-11 09:22:38,036 xinference.model.llm.vllm.core 1025864 INFO     Loading qwen2.5-instruct with following model config: {'max_model_len': 32760, 'tokenizer_mode': 'auto', 'trust_remote_code': True, 'tensor_parallel_size': 1, 'block_size': 16, 'swap_space': 4, 'gpu_memory_utilization': 0.9, 'max_num_seqs': 256, 'quantization': None, 'limit_mm_per_prompt': None}Enable lora: False. Lora count: 0.
2024-10-11 09:22:38,040 xinference.core.worker 1025075 ERROR    Failed to load model qwen2.5-instruct-1-0
Traceback (most recent call last):
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/worker.py", line 893, in launch_builtin_model
    await model_ref.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 231, in send
    return self._process_result_message(result)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 102, in _process_result_message
    raise message.as_instanceof_cause()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 656, in send
    result = await self._run_coro(message.message_id, coro)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 367, in _run_coro
    return await coro
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/api.py", line 384, in __on_receive__
    return await super().__on_receive__(message)  # type: ignore
  File "xoscar/core.pyx", line 558, in __on_receive__
    raise ex
  File "xoscar/core.pyx", line 520, in xoscar.core._BaseActor.__on_receive__
    async with self._lock:
  File "xoscar/core.pyx", line 521, in xoscar.core._BaseActor.__on_receive__
    with debug_async_timeout('actor_lock_timeout',
  File "xoscar/core.pyx", line 526, in xoscar.core._BaseActor.__on_receive__
    result = await result
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/model.py", line 309, in load
    self._model.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/model/llm/vllm/core.py", line 251, in load
    engine_args = AsyncEngineArgs(
TypeError: [address=0.0.0.0:37737, pid=1025864] AsyncEngineArgs.__init__() got an unexpected keyword argument 'limit_mm_per_prompt'
2024-10-11 09:22:38,085 xinference.core.worker 1025075 ERROR    [request 481f598e-876f-11ef-bd92-3cecefab736b] Leave launch_builtin_model, error: [address=0.0.0.0:37737, pid=1025864] AsyncEngineArgs.__init__() got an unexpected keyword argument 'limit_mm_per_prompt', elapsed time: 8 s
Traceback (most recent call last):
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/utils.py", line 69, in wrapped
    ret = await func(*args, **kwargs)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/worker.py", line 893, in launch_builtin_model
    await model_ref.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 231, in send
    return self._process_result_message(result)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 102, in _process_result_message
    raise message.as_instanceof_cause()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 656, in send
    result = await self._run_coro(message.message_id, coro)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 367, in _run_coro
    return await coro
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/api.py", line 384, in __on_receive__
    return await super().__on_receive__(message)  # type: ignore
  File "xoscar/core.pyx", line 558, in __on_receive__
    raise ex
  File "xoscar/core.pyx", line 520, in xoscar.core._BaseActor.__on_receive__
    async with self._lock:
  File "xoscar/core.pyx", line 521, in xoscar.core._BaseActor.__on_receive__
    with debug_async_timeout('actor_lock_timeout',
  File "xoscar/core.pyx", line 526, in xoscar.core._BaseActor.__on_receive__
    result = await result
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/model.py", line 309, in load
    self._model.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/model/llm/vllm/core.py", line 251, in load
    engine_args = AsyncEngineArgs(
TypeError: [address=0.0.0.0:37737, pid=1025864] AsyncEngineArgs.__init__() got an unexpected keyword argument 'limit_mm_per_prompt'
2024-10-11 09:22:38,099 xinference.api.restful_api 1024902 ERROR    [address=0.0.0.0:37737, pid=1025864] AsyncEngineArgs.__init__() got an unexpected keyword argument 'limit_mm_per_prompt'
Traceback (most recent call last):
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/api/restful_api.py", line 967, in launch_model
    model_uid = await (await self._get_supervisor_ref()).launch_builtin_model(
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 231, in send
    return self._process_result_message(result)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 102, in _process_result_message
    raise message.as_instanceof_cause()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 656, in send
    result = await self._run_coro(message.message_id, coro)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 367, in _run_coro
    return await coro
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/api.py", line 384, in __on_receive__
    return await super().__on_receive__(message)  # type: ignore
  File "xoscar/core.pyx", line 558, in __on_receive__
    raise ex
  File "xoscar/core.pyx", line 520, in xoscar.core._BaseActor.__on_receive__
    async with self._lock:
  File "xoscar/core.pyx", line 521, in xoscar.core._BaseActor.__on_receive__
    with debug_async_timeout('actor_lock_timeout',
  File "xoscar/core.pyx", line 526, in xoscar.core._BaseActor.__on_receive__
    result = await result
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/supervisor.py", line 1032, in launch_builtin_model
    await _launch_model()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/supervisor.py", line 996, in _launch_model
    await _launch_one_model(rep_model_uid)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/supervisor.py", line 975, in _launch_one_model
    await worker_ref.launch_builtin_model(
  File "xoscar/core.pyx", line 284, in __pyx_actor_method_wrapper
    async with lock:
  File "xoscar/core.pyx", line 287, in xoscar.core.__pyx_actor_method_wrapper
    result = await result
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/utils.py", line 69, in wrapped
    ret = await func(*args, **kwargs)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/worker.py", line 893, in launch_builtin_model
    await model_ref.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 231, in send
    return self._process_result_message(result)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/context.py", line 102, in _process_result_message
    raise message.as_instanceof_cause()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 656, in send
    result = await self._run_coro(message.message_id, coro)
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/backends/pool.py", line 367, in _run_coro
    return await coro
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xoscar/api.py", line 384, in __on_receive__
    return await super().__on_receive__(message)  # type: ignore
  File "xoscar/core.pyx", line 558, in __on_receive__
    raise ex
  File "xoscar/core.pyx", line 520, in xoscar.core._BaseActor.__on_receive__
    async with self._lock:
  File "xoscar/core.pyx", line 521, in xoscar.core._BaseActor.__on_receive__
    with debug_async_timeout('actor_lock_timeout',
  File "xoscar/core.pyx", line 526, in xoscar.core._BaseActor.__on_receive__
    result = await result
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/core/model.py", line 309, in load
    self._model.load()
  File "/home/test_wj/anaconda3/envs/xinfer/lib/python3.10/site-packages/xinference/model/llm/vllm/core.py", line 251, in load
    engine_args = AsyncEngineArgs(
TypeError: [address=0.0.0.0:37737, pid=1025864] AsyncEngineArgs.__init__() got an unexpected keyword argument 'limit_mm_per_prompt'

```

### Expected behavior / 期待表现

successfully load the model

**Comment by zjuyzj** [comment]
There seems to be some dependency problem after PR #2384, since argument `limit_mm_per_prompt` is not available until vllm v0.5.5. Upgrading your vllm with `pip install vllm>=0.5.5 --upgrade` may help.

**Comment by amumu96** [comment]
We will fix at next release.
